workflow/lib/shared/initialize_paramsearch.py

```diff
@@ -0,0 +1,173 @@
+"""Initialize parameter searches."""
+
+from snakemake.utils import Paramspace
+import pandas as pd
+from itertools import product
+from pathlib import Path
+
+from lib.shared.file_utils import get_filename
+
+def initialize_segment_sbs_paramsearch(config):
+    """Initialize parameter search for sbs segmentation.
+
+    Args:
+        config (dict): Configuration dictionary.
+
+    Returns:
+        dict: Updated configuration dictionary.
+        list: Nuclei diameters.
+        list: Cell diameters.
+        list: Flow thresholds.
+        list: Cell probabilities.
+        dict: Output patterns.
+
+    """
+
+    if config['sbs_process'].get('mode') != 'segment_sbs_paramsearch':
+        return config, None, None, None, None
+        
+    if 'paramsearch' not in config['sbs_process']:
+        base_nuclei = config['sbs_process']['nuclei_diameter']
+        base_cell = config['sbs_process']['cell_diameter']
+        nuclei_diameters = [base_nuclei - 2, base_nuclei, base_nuclei + 2]
+        cell_diameters = [base_cell - 2, base_cell, base_cell + 2]
+        flow_thresholds = [0.2, 0.4, 0.6]
+        cell_probs = [-4, -2, 0, 2, 4]
```

Comment:
We should probably be defining constants for this type of thing correct? Maybe at the top of this file or in another file? This will give any developers context on these numbers and will make changing these numbers more clear what will happen.